packages/frontend/src/components/create-campaign/form/start-date-step/index.tsx

```diff
@@ -39,6 +52,31 @@ export function StartDateStep({
         setOpen(true);
     }, [disabled]);
 
+    useEffect(() => {
+        if (minDate || !open) return;
+        setMinDate(getClosestAvailableDateTime());
+    }, [minDate, open]);
+
+    useEffect(() => {
+        const interval = setInterval(() => {
+            if (date?.isBefore(dayjs()))
+                setMinDate(getClosestAvailableDateTime(date));
+        }, 1_000);
+        return () => {
+            clearInterval(interval);
+        };
+    }, [date, minDate]);
+
+    useEffect(() => {
+        if (!date || !minDate || !previousDate) return;
```

Comment:
I think we need to run this check every second or something. I remember we had the same "issue" with Carrot.